### PR TITLE
Complete load script

### PR DESCRIPTION
**Pull request by ieva7**
Fully functional
Checks against database for duplicates
Loads data to RDS
I had git rebase issues so no clue what files are up for change

### CODE REVIEW

File: pipeline/load.py [hunk 1]
```diff
@@ -31,42 +48,150 @@ def get_data_from_db(conn: connection, table: str, column = "*")-> pd.DataFrame:
...
+
+    with conn.cursor() as cur:
+        tuples = df.to_records(index=False)
+        execute_values(cur, """INSERT INTO author (author_name) VALUES %s;""", tuples)
+        conn.commit()
+
+
+def add_to_article_version_table(conn: connection, df: pd.DataFrame) -> None:
+    """Converts df into tuples, then adds to article_version table.
+    NB: needs article_id column converted into foreign key reference"""
```
Comment (jGreeman) [inline pipeline/load.py:139]: is this note still needed?

File: pipeline/load.py [hunk 2]
```diff
@@ -31,42 +48,150 @@ def get_data_from_db(conn: connection, table: str, column = "*")-> pd.DataFrame:
...
+def check_for_duplicate_authors(conn: connection, name: str) -> str:
+    """Checks for a duplicates in author table. Returns a dataframe without duplicates."""
 
-    return df
+    with conn.cursor() as cur:
+        cur.execute("""SELECT * FROM author where author_name = %s;""", [name])
+        duplicate = cur.fetchall()
+        if len(duplicate) > 0:
+            return None
+    return name
```
Comment (jGreeman) [inline pipeline/load.py:70]: Are these functions making sure we don't add articles with the same id twice or authors with the same name twice?
Could it be easier to make these columns unique in the DB schema?

Comment (jGreeman) [reply pipeline/load.py:70]: All looks v good though :)

File: pipeline/load.py (hunk 1 above)
Comment (ieva7) [reply pipeline/load.py:139]: Do you mean the docstring? Because the article_id is a foreign key so it's needed as a disclaimer

Comment (jGreeman) [reply pipeline/load.py:139]: ah okay I thought it was like a TODO later type thing, nvm

File: pipeline/load.py (hunk 2 above)
Comment (ieva7) [reply pipeline/load.py:70]: If we did a UNIQUE constraint, I think then we'd have to do a try:except loop and still log which articles were rejected - i think my way *might* be faster or the same

Comment (ieva7) [reply pipeline/load.py:70]: The two functions each check for article and for author tables respectively also

Comment (jGreeman) [reply pipeline/load.py:70]: Couldn't we do a ON CONFLICT DO NOTHING statement like we did last week, do we need to log the rejections?

Comment (jGreeman) [reply pipeline/load.py:70]: okay cool idm either way, it's very minor

**Pull request merged**